metaflow/plugins/argo/argo_workflows.py

```diff
@@ -72,6 +89,7 @@ def __init__(
         max_workers=None,
         workflow_timeout=None,
         workflow_priority=None,
+        ignore_events=False,
```

Comment:
the CLI should be able to determine whether the event backend is configured or not and pass `ignore_events=True` if the event backend is missing? that will remove the need for `are_events_configured` method's usage in this file.

Reply:
Will fix